authorizer-app/src/index.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8">
   <title>AuthorizerApp</title>
-  <base href="/rest-sources/authorizer/">
+  <base href="/">
```

Comment:
I think you have to revert this for docker build

Reply:
I added base href as an argument to the docker build so the path should be added then i think

Reply:
I changed it back but can still be controlled by docker ARGS